core/src/main/java/nu/yona/server/properties/SecurityProperties.java

```diff
@@ -155,4 +156,14 @@ public void setOvpnProfileFile(String ovpnProfileFile)
 	{
 		this.ovpnProfileFile = ovpnProfileFile;
 	}
+
+	public String getFirebaseAdminServiceAccountKeyFile()
```

Comment:
Above, I proposed to add an ``isEnabled`` property for Firebase. For consistency with services like e-mail and LDAP, you might want to add a class ``FirebaseProperties`` with three properties: ``isEnabled``, ``adminServiceAccountKeyFile`` and ``databaseUrl``.

Reply:
Hmm, that indeed cleans it up